code/game/objects/items/plushes.dm

```diff
@@ -542,3 +542,133 @@
 	attack_verb = list("stung")
 	gender = FEMALE
 	squeak_override = list('sound/voice/moth/scream_moth.ogg'=1)
+
+/obj/item/toy/plush/goatplushie
+	name = "strange goat plushie"
+	icon_state = "goat"
+	desc = "Despite its cuddly appearance and plush nature, it will beat you up all the same. Goats never change."
+
+/obj/item/toy/plush/goatplushie/angry
+	var/mob/living/carbon/target
+	throwforce = 6
+	var/cooldown = 0
+	var/cooldown_modifier = 20
+
+/obj/item/toy/plush/goatplushie/angry/Initialize()
+	. = ..()
+	START_PROCESSING(SSprocessing, src)
+
+/obj/item/toy/plush/goatplushie/angry/process()
+	if (prob(25) && !target)
+		var/list/targets_to_pick_from = list()
+		for(var/mob/living/carbon/C in view(7, src))
```

Comment:
make sure that the goat plushie(s) won't attack members of the goat faction (or target them; you'll need two checks, one here and one down in ram() (in case someone puts on a goat pelt hat or the like after being targeted but before being attacks)